utils/ansible-runner-service.sh

```diff
@@ -416,7 +416,7 @@ check_access() {
 manage_ansible_hosts_file() {
     echo "Linking the runner service inventory to ceph-ansible hosts"
     # create hosts file if it doesn't exist
-    [ ! -f $CEPH_ANSIBLE_HOSTS ] && touch $CEPH_ANSIBLE_HOSTS
+    [ ! -s $CEPH_ANSIBLE_HOSTS ] && touch $CEPH_ANSIBLE_HOSTS
```

Comment:
this isn't what I was talking about. this should just be file exists (-f) and please use the variable in quotes, then see below